### PR TITLE
Create subscription from USSD flow

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Same as SMS: https://github.com/praekelt/go-usaid-mmc/blob/develop/src/sms_app.js#L131

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps review please

File: go-app-ussd.js [hunk 1]
```diff
@@ -249,13 +249,13 @@ go.utils = {
 
         subscription_subscribe: function(contact, im) {
             var payload = {
-              contact_key: contact.key,
-              lang: 'en',
```
Comment (imsickofmaps) [inline go-app-ussd.js]: given we ask what lang for USSD we should just set it correctly

File: src/utils.js [hunk 2]
```diff
@@ -242,13 +242,13 @@ go.utils = {
...
             var payload = {
-              contact_key: contact.key,
-              lang: 'en',
-              message_set: "/subscription/api/v1/message_set/12/",
-              next_sequence_number: 2,
-              schedule: "/subscription/api/v1/periodic_task/1/",
-              to_addr: contact.msisdn,
-              user_account: contact.user_account
+                contact_key: contact.key,
+                lang: 'en',
```
Comment (imsickofmaps) [inline src/utils.js]: given we ask what lang for USSD we should just set it correctly

**Comment by imsickofmaps** [comment]
Just one thing, let me check it before landing

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps check before landing oO

**Comment by imsickofmaps** [comment]
@NicolaasDieBaas changes are fine but lets add the change lang update the subscription before landing

**Comment by imsickofmaps** [comment]
the fix from #22 is needed though, so don't just use the current function

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps have a final look please

**Comment by imsickofmaps** [comment]
👍  when travis happy

**Pull request merged**